packages/fx-core/tests/common/local/localEnvManager.test.ts

```diff
@@ -214,7 +214,7 @@ describe("LocalEnvManager", () => {
         name: "fx-solution-azure",
         hostType: "Azure",
         capabilities: ["Tab"],
-        activeResourcePlugins: ["fx-resource-simple-auth"],
+        activeResourcePlugins: ["fx-resource-simple-auth", "fx-resource-frontend-hosting"],
```

Comment:
Why changing test data here? Are there new restrictions on `activeResourcePlugins`?

Reply:
Migration from v2 to v3 depends on "activeResourcePlugins" instead of "capabilities".